src/components/FileTreeNode/index.tsx

```diff
@@ -101,13 +105,22 @@ class FileTreeNodeBase<TreeNodeType> extends React.Component<PublicProps> {
     let linterType = null;
     if (linterMessages && hasLinterMessages) {
       linterType = findMostSevereTypeForPath(linterMessages, node.id);
+      let title = getTitleForType(linterType, isFolder);
+
+      if (
+        linterType === 'notice' &&
```

Comment:
I believe if the most severe type is `notice`, we can semi-hide the file, no? Or should we make sure that there is only/exactly one message?